### PR TITLE
DXCDT-311: Editorializing "hints"

### PR DESCRIPTION
**Pull request by willvedd**
### 🔧 Changes

Combing through all the hints and modifying some of the language used. For example, changing "you may wanna..." to "consider running...". Also added a hint after logging in with multiple tenants.

### 📝 Checklist

- [x] All new/changed/fixed functionality is covered by tests (or N/A)
- [x] I have added documentation for all new/changed functionality (or N/A)

### CODE REVIEW

File: internal/cli/login.go [hunk 1]
```diff
@@ -122,6 +122,12 @@ auth0 login --scopes "read:client_grants,create:client_grants"`,
 
 			cli.tracker.TrackCommandRun(cmd, cli.config.InstallID)
 
+			if len(cli.config.Tenants) > 1 {
+				cli.renderer.Infof("%s switch between authenticated tenants with `auth0 tenants use <TENANT>`",
```
Comment (sergiught) [inline internal/cli/login.go:126]:
```suggestion
				cli.renderer.Infof("%s switch between authenticated tenants with `auth0 tenants use <tenant>`",
```

File: internal/cli/test.go [hunk 2]
```diff
@@ -183,7 +183,8 @@ auth0 test login <client-id> --connection <connection>`,
 			}
 
 			if isFirstRun {
-				cli.renderer.Infof("%s Login flow is working! Next, try downloading and running a Quickstart: 'auth0 quickstarts download %s'",
+				cli.renderer.Infof("Login flow is working!")
+				cli.renderer.Infof("%s Consider downloading and running a quickstart by running `auth0 quickstarts download %s`",
```
Comment (sergiught) [inline internal/cli/test.go:187]:
```suggestion
				cli.renderer.Infof("%s Consider downloading and running a quickstart next, by running `auth0 quickstarts download %s`",
```

Wdyt? Personally I find this a bit nicer, the user logged in the first time, let's suggest what to do next.

File: internal/cli/universal_login.go [hunk 3]
```diff
@@ -353,7 +353,7 @@ func (cli *cli) customTemplateEditorPromptWithPreview(cmd *cobra.Command, body *
 	defer cancel()
 
 	onInfo := func() {
-		cli.renderer.Infof("%s once you close the editor, you'll be prompted to save your changes. To cancel, CTRL+C.", ansi.Faint("Hint:"))
+		cli.renderer.Infof("%s once you close the editor, you'll be prompted to save your changes. To cancel, press CTRL+C.", ansi.Faint("Hint:"))
```
Comment (Widcket) [inline internal/cli/universal_login.go:356]: Some hints are sentence-cased following the "Hint:", other begin with a lowercase letter. Should we stick to a single style?

Comment (willvedd) [reply internal/cli/universal_login.go]: Good catch, we decided to upper-case them all 👍

**Review by sergiught — APPROVED** [review]

**Pull request merged**